tests/execute/result/check.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "run=\$(mktemp -d)" 0 "Create run directory"
+        rlRun "PROVISION_HOW=${PROVISION_HOW:-virtual}"
+        rlRun "pushd check"
+        rlRun "set -o pipefail"
+        # Write pattern for tests that need pre-existing dmesg content
+        rlRun "echo 'Fail Test Check Pattern' | sudo tee /dev/kmsg"
```

Comment:
Actual check tests are run on a guest provisioned on line 15, by default `virtual` provision is used, so this line will have no effect.